scriptkeeper-in-docker.sh.test.yaml

```diff
@@ -1,5 +1,5 @@
 unmockedCommands:
-  - /usr/bin/basename
+  - basename
```

Comment:
I noticed that you can't specify `unmockedCommands` as absolute paths anymore after this change. I think this test file was written when the executable shortform wasn't supported yet. So I still think this is a good change. @hallettj